### PR TITLE
Fix/images

### PR DESCRIPTION
**Pull request by mmoila**
- Api requests to absolute paths
- Fix mixed alt texts
- Display thanks page picture even without alt text
- Improve thanks page styling

### CODE REVIEW

File: client/src/components/MapQuestion.tsx [hunk 1]
```diff
@@ -61,7 +61,7 @@ export default function MapQuestion({ value, onChange, question }: Props) {
         valueRef.current.map((answer, index) => ({
           ...answer,
           geometry: features[index],
-        })),
```
Comment (jlaamanen) [inline client/src/components/MapQuestion.tsx:64]: [nit] These trailing commas should stay here according to our Prettier config, wonder if you have the correct auto-formatting rules in your editor?

**Review by jlaamanen — APPROVED** [review]
LGTM 🌟

File: client/src/components/SurveyThanksPage.tsx [hunk 2]
```diff
@@ -73,22 +73,30 @@ export default function SurveyThanksPage({ survey, isTestSurvey }: Props) {
           {' '}
           <img
             style={{ height: '125px', width: '351px' }}
-            src={`api/feature-styles/icons/tre_logo`}
+            src={`/api/feature-styles/icons/tre_logo`}
             alt={tr.IconAltTexts.treLogoAltText}
           />
         </div>
-        <Typography variant="h5">
+        <Typography variant="h5" mt={survey.thanksPage.imageName ? 10 : 0}>
```
Comment (jpeltzi) [inline client/src/components/SurveyThanksPage.tsx:80]: Add `component="h1"` to have the heading render as `h1` but look like `h5`.

**Review by jpeltzi — CHANGES_REQUESTED** [review]
Two semantics issues that need attention, see comments.
The `<div>` wrapper solution is a bit hacky but it shall do for now.

**Review by jpeltzi — APPROVED** [review]

**Pull request merged**